gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -68,6 +68,10 @@ if CLIENT then
 	-- if the files are different, it only looks at the name. I recommend you
 	-- create your own directory so that this does not happen,
 	-- eg. /materials/vgui/ttt/mycoolserver/mygun.vmt
+
+	-- The ViewModel should not draw, in any situation. Prevents the need for hacks which
+	-- overload drawing functions.
+	SWEP.InvisibleViewModel = false
```

Comment:
Which hacks are you talking about? Can you give an example?

Reply:
hacks as in overriding DrawViewModel in the deploy (only works once), setting the FOV < 10 (the viewmodel still exists, just out of frame)

Reply:
I still need a real example like is this even done anywhere? I would assume to see a deletion of such a hacky way in the code when you say you removed the need for it. But i cant find anything

Reply:
[yes, but i don't have any suggestions for what FOV an invisible viewmodel should be, so I didn't change it](https://github.com/search?q=repo%3ATTT-2%2FTTT2%20%22ViewModelFOV%20%3D%2010%22&type=code), the rest of the [DrawViewModel changes are in the changelist](https://github.com/TTT-2/TTT2/pull/1311/files)

Reply:
> I still need a real example like is this even done anywhere?

The unarmed weapon does this, with my cam rework you are sometimes able to see the crowbar model